app/Http/Controllers/FrontendTransportController.php

```diff
@@ -125,7 +125,7 @@ public function TrainTrip(Request $request): Renderable|RedirectResponse {
     public function TrainCheckin(Request $request): RedirectResponse {
         $this->validate($request, [
             'body'           => 'max:280',
-            'business_check_new_travel' => 'digits_between:0,2',
+            'business_check' => 'digits_between:0,2',
```

Comment:
Warum hast du die umbenannt? Es gibt ja den bussines checkin und den für bei einer neuen Reise, so sind die Variablenen Namen einduetiger?

Reply:
Ich hab die umbenannt, weil ich beide Modals in eine extra Datei ausgelagert habe und im Backend (nicht in der Validation) die variablen in beiden Fällen businessCheck heißen. Der Bedeutungsunterschied liegt dann nur noch in der Route und nicht mehr in den Variablen. :)